index.html

```diff
@@ -208,22 +234,29 @@ <h2>How does it work?</h2>
 
       // Create a Sanitizer with that configuration:
       let sanitizer = null;
-      try {
-        const conf = JSON.parse(configText.value);
-        sanitizer = new Sanitizer(conf);
-      } catch (err) {
-        replace(outputHtml, "warnparseconfig");  // Display a parse error.
-        return;
+      if (config !== "default") {
```

Comment:
With the new API, we now support the string "default" as a config. Maybe we can drop the if (and same in line 251), in favour of something like:

```
try {
  const conf = (config === default) ? config : JSON.parse(configText.value);
  ...
```

All subsequen code should then work without any additional checks.